--- kdbx/kdbx/kdbx.desktop~	2005-05-13 11:36:02.000000000 +0200
+++ kdbx/kdbx/kdbx.desktop	2005-05-13 11:37:17.000000000 +0200
@@ -1,12 +1,13 @@
 # KDE Config File  
 [Desktop Entry]
+Encoding=UTF-8
 Type=Application
 Exec=kdbx -caption "%c" %i %m  
 Icon=kdbx.png
 MiniIcon=kdbx.png
 DocPath=kdbx/index.html
-Comment=
-Comment[de]=
-Terminal=0
+Comment=Loading folder structure from Outlook Express 5.5 directory
+Comment[pl]=Wczytywanie struktury folderów z katalogu Outlook Expressa 5.5
+Terminal=false
 Name=Kdbx
-Name[de]=Kdbx
+Categories=Qt;KDE;Utility;
